src/components/control/select.jsx

```diff
@@ -195,6 +196,30 @@ const Select = forwardRef(function Select(props, ref) {
       .forEach(_ref => _ref.current.setSelected(JSON.stringify(value) === JSON.stringify(_ref.current.value)));
   }, [value, showOptions]);
 
+  useEffect(() => {
```

Comment:
If you are manipulating the DOM (e.g. `focus`) then you _need_ to use `useLayoutEffect`

Reply:
I've had issues with `useLayoutEffect` but i'll give that a shot.